script/Ciel/Day 14/14_01B_03_CIEL14B_4_1E.txt

```diff
@@ -86,197 +97,248 @@ What a joke.
+// Banished to the goddamn phantom zone
+Instead of being dashed to pieces against the ground,
+ the Executor will be transformed into a 'planar event',
+ deprived of their freedom,
+ and forced to suffer for all eternity.
+ A fate which offers no chance at salvation.
 }
 [sha:423038222e88962090cf18eeffa158544709d964]{
 -- Page 116, Offset 40585.
 -- 光体となった彼女の予測は絶対である。
-A prediction coming from a Luminary like herself is absolute.
+Now that she has become a Luminary,
+ her predictions are absolute.
 }
 [sha:5cf047bcc574c60cab5dd6bc1b06f9f7c3b22214]{
 -- Page 116, Offset 40586.
 -- 代行者シエルの運命は、
-The fate of Executor Ciel...
+The fate of Executor Ciel―――
 }
 [sha:0c94c2c0f1239d741266b1bd32f80917e4185466]{
 -- Page 116, Offset 40587.
 -- あの人間……否、転生者ロアの行動は、
-The actions of that human... no, of Roa's reincarnation,
+The actions of that human――no, of Roa's reincarnation―――
 }
 [sha:ec772953449ee3743b05318c7e73954f2e724c68]{
 -- Page 116, Offset 40588.
 -- 彼女の思うままでなければならない。
-must be persecuted to her heart's will.
+They are all destined to occur as she wills it.
 }
-[sha:21652fae76fe1a32551860d9fb35bdf485cc48f3]{
+[offset:40589]{
 -- Page 117, Offset 40589. Mods: @n.
 -- そう。
-That's right.
+That's right―――
 }
 [sha:9446b5ab9abd601c6978aa1bfe7f933668e0267a]{
 -- Page 117, Offset 40590.
 -- 全能者が予測を外す事はあってはならない。
-The prediction of the almighty can never be wrong.
+The predictions of the Almighty can never be wrong.
 }
 [sha:c926c70f616035a8994e1b8ee216e0405f951ae1]{
 -- Page 117, Offset 40591.
 -- それだけの能力、それだけの知覚を持っているのだ。
-That's the kind of ability and perception that she possesses.
+That's just how powerful she is.
+ How all-encompassing her perception is.
 }
 [sha:b59a60f105e6c84258e129feb3e563c61c8439b9]{
 -- Page 117, Offset 40592.
 -- もし外す事があれば、それは世界への侮辱であり、自身の無能さを証明する事に他ならない。
-If that prediction happens to be wrong, then it would be an insult to the world and be nothing but proof of her own incompetence.
+If her predictions were to be incorrect,
+ it would be an insult to the world itself;
+ nothing short of irrefutable proof of her own incompetence.
 }
-[sha:ee4882edb7558238ce45737a587685707e65cda4]{
+[offset:40593]{
 -- Page 117, Offset 40593.
 -- なので。
-But despite that...
+And yet―――
 }
 [sha:afe17b147716bb3344fe9851c60a2ae39b1671ef]{
 -- Page 118, Offset 40594. Mods: @n.
 -- 「“え―――
-"Eh―――
+"Huh―――
 }
 [sha:fd0584bbb6c191474866bc4a04f55175c945b598]{
 -- Page 118, Offset 40595. Glued.
 -- えぇぇええ！？”」
-eeeeeh?!"
+whaaaaaa?!"
 }
 [sha:2ebbb7f08403e350b121c7ca0d78b9d83b8485b4]{
 -- Page 118, Offset 40596.
 -- いま目の前で起きている展開は、彼女にとって、光体にまで変革した自分の立場を忘れるほどの衝撃だった。
-The development that unfolded before her eyes was a shock which made her completely forget her own position as a Luminary.
+The events that just transpired before her eyes come as such a shock that she completely forgets that she's a Luminary.
```

Comment:
Something like this
光体にまで変革した自分の立場 is significant, because she forgets "her place" and starts moving because of what she sees here.
>What is currently transpiring before her eyes comes as such a shock that she completely forgets what her obligations after having reformed into a Luminary are.